src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -2522,12 +2528,15 @@ private void printDevModeMessages(boolean inputUnavailable, boolean startup) thr
     }
 
     private void printTestsMessage(boolean formatForAttention) {
-        if (hotTests) {
-            String message = "Tests will run automatically when changes are detected. You can also press the Enter key to run tests on demand.";
-            info(formatForAttention ? formatAttentionMessage("Enter - " + message) : message);
-        } else {
-            String message = "run tests on demand, press Enter.";
-            info(formatForAttention ? formatAttentionMessage("Enter - " + message) : "To " + message);
+        // setting skipTests to true overrides the setting of hotTests, and prevents any tests from running in dev mode (automatically or on demand)
+        if (!skipTests) {
```

Comment:
In a similar line of thought to manage nested conditions, this could also be expressed as
``` 
if (skipTests) {
    return;
}
```

Reply:
This one I wouldn't mind changing since it doesn't affect the structure of the rest of the method. I'm not going to run through all the PR testing for LMP/LGP again though if I change this.